tables/gurumuki.cti

```diff
@@ -1,7 +1,13 @@
 # This table contains braille codes and rules for Gurumukhi script.
-#
-# Copyright (C) 2014 National Institute for Visually Handicapped, 116, Rajpur Road, Dehradun, Uttrakhand, India
-#
+
+#-author-name: Dipendra Manocha
+#-author-email: dmanocha@daisy.org
+#-maintainer-name: Dipendra Manocha
+#-maintainer-email: dmanocha@daisy.org
+#-updated: 2022-05-23
+
+# Copyright (C) 2022 National Institute for Visually Handicapped, 116, Rajpur Road, Dehradun, Uttrakhand, India
```

Comment:
All years in which changes were made should be included in the copyright line:

> Copyright (C) 2014,2022 National Institute for Visually Handicapped, 116, Rajpur Road, Dehradun, Uttrakhand, India

And why did you remove your line?

> Copyright (C) 2021 by Compass Braille <http://www.compassbraille.org>

Reply:
Sorry I am not really familiar with copyright law etc. I will amend but I think Compass Braille year should be 2022? (I think I had 2021 was because I made some changes last year but didn't get round to submitting them to liblouis).

Reply:
I think in this case either "2022" or "2021-2022" is fine.